ports/servoshell/app.rs

```diff
@@ -122,8 +123,8 @@ impl App {
         }
 
         if let Some(mut minibrowser) = app.minibrowser() {
-            minibrowser.update(window.winit_window().unwrap(), "init");
-            window.set_toolbar_height(minibrowser.toolbar_height.get());
+            minibrowser.update(window.winit_window().unwrap(), None, "init");
```

Comment:
Why is the framebuffer None here? Might be nice to have a little comment explaining why.

Reply:
done in 30944bd054323bbcf42e106dcd9256ebef030542